fem/fespace.cpp

```diff
@@ -1517,6 +1519,42 @@ void FiniteElementSpace::UpdateNURBS()
    ndofs = NURBSext->GetNDof();
    elem_dof = NURBSext->GetElementDofTable();
    bdrElem_dof = NURBSext->GetBdrElementDofTable();
+   delete face_dof;
+   face_dof = NULL;
+}
+
+void FiniteElementSpace::GenerateFaceDofsFromBdr()
```

Comment:
Calling this public function on a non-NURBS finite element space results in a segfault. It might be good to at least add a `MFEM_VERIFY` check to give an error message.

Reply:
Do you mean the `UpdateNURBS` function? Or the `GenerateFaceDofsFromBdr`?
The latter should work for non-NURBS??? ALthough I did not check, and don;t directly see a purpose. For the former I will add it for safetly.

Reply:
I meant `GenerateFaceDofsFromBdr`. For a non-NURBS space, `bdrElem_dof` will be NULL, and so it will crash on line 1535.

Reply:
Instead of accessing the `bdrElem_dof `table, I now make a call to` GetBdrElementDofs`, this should work for all meshes.

Reply:
It still segfaults for me when you access `bdrElem_dof` on line 1538.

Reply:
Correct. Replaced access to `bdrElem_dof` and `mesh` for sizes info with appropriate `Get` function.